apps/git.go

```diff
@@ -307,9 +275,15 @@ func (fs *gfs) ReadDir(dirname string) ([]gitFS.FileInfo, error) {
 
 func (fs *gfs) TempFile(dirname, prefix string) (gitFS.File, error) {
 	// TODO: not really robust tempfile...
```

Comment:
For the record, this is how `TempFile` is implemented in the stdlib: https://golang.org/src/io/ioutil/tempfile.go?s=1285:1342#L37